src/database/triggers.sql

```diff
@@ -8,3 +8,22 @@ $func_create_user$ LANGUAGE plpgsql security definer;
 DROP trigger if exists trigger_create_user on auth.users;
 CREATE TRIGGER trigger_create_user after insert on auth.users
     FOR EACH ROW EXECUTE PROCEDURE func_create_user();
+
+
+CREATE OR REPLACE FUNCTION func_match_notification() RETURNS trigger AS $func_match_notification$
+    BEGIN
+        update public.users
+        set "matchNotification" = "matchNotification" + 1
```

Comment:
I have added a column called matchNotification in public.users table.
This attribute is updated by a trigger each time an insert in match is performed.
The update affects the users concerned by the new match relationship.
Then, a listener (websocket) is installed in useDatabase.ts/useUserId/matchInsert to listen for updates on this counter.
That way, we can react to a new match for a specific user and update the match list accordingly.
This whole trick was implemented because of the lack of filtering for insert listeners with supabase.
Long story short : you can't listen for an insert event for one and only one user because, well, it is not in the db yet...
Listening for changes in the match table and then filtering for the logged in user would be performance heavy if lots of matches happen at the same time across our future 10k + user base 💯